Source/Cogworks.Umbraco.Essentials/Extensions/ImageCropperExtensions.cs

```diff
@@ -8,9 +8,28 @@ namespace Cogworks.Umbraco.Essentials.Extensions
 {
     public static class ImageCropperExtensions
     {
-        public static string GetCropUrls(this IPublishedContent image, string cropAlias, int? width = null, int? height = null, bool includeRetina = true, bool enableWebP = false, int? quality = null)
+        public static string GetCropUrls(this IPublishedContent image, string cropAlias = null, int? width = null, int? height = null, bool includeRetina = true, bool enableWebP = false, int? quality = null, bool returnOnlyRetina = false)
```

Comment:
IMO we should move/split those parameters to new line, they are to long in single line.

Reply:
Or we could think to create some configuration class as parameter that contains those settings there.
```
public static string GetCropUrls(this IPublishedContent image, CropOptions options)
```